pennylane/pulse/__init__.py

```diff
@@ -220,7 +220,7 @@ def circuit(params):
         qml.evolve(H)(params, t=[0, 10])
         return qml.expval(qml.PauliZ(0))
 
->>> params = [1.2]
+>>> params = jnp.array([1.2])
```

Comment:
Why?

Reply:
For context, I don't think we should encourage users to make `params` a `jnp.array` in examples. Doing so for any Hamiltonian where different callables take a different number of parameters will raise an error. The `params` argument passed to a `ParametrizedHamiltonian` should be a list or tuple - items in the list might be arrays, but `params` is not one.

Reply:
Ah, this is just to match the provided output. The other way around, I can make the output a list of a 0d array :)

Reply:
Perfect, updating output sounds better I think!